src/sbt-test/interplay/plugin-with-root/test

```diff
@@ -17,14 +17,14 @@ $ exec git branch -u origin/main
 # However, that initial state does not contain the above created .git folder (created via git init) yet.
 > reload
 
-> release cross with-defaults
```

Comment:
The reason why I removed `cross` here is because IMHO this is not a real life use case. This test here, as its name `plugin-with-root` tells us, tests a sbt plugin that activates the `PlayRootProject` plugin (see `PlayRootProjectBase.scala` above). As you can see above, the `PlayRootProject` did set crossScalaVersion to 2.12 until now. Now, I changed that to 2.13, which makes the scripted test here fail because it can not resolve `.../org/scala-sbt/scripted-sbt_2.13/1.5.5/...`
That is, as we know, because sbt plugins for sbt 1 need to be published with scala 2.12.
I had a look at the projects that still use interplay (#150) and which, from these projects, make use of the `PlayRootProject` plugin. That are
* https://github.com/playframework/scalatestplus-play/blob/master/build.sbt#L76
* https://github.com/playframework/play-slick/blob/master/build.sbt#L30
* https://github.com/playframework/playframework/blob/f18c239c9186b5b65af291817cd75918f5d39ff9/build.sbt#L470

Now it turns out that all of these projects explicitly set `crossScalaVersions` anyway, overriding the value set by `PlayRootProject`:
* https://github.com/playframework/scalatestplus-play/blob/e74e29b9e5a1ae001b1db8b218a4666e52321596/build.sbt#L69
* https://github.com/playframework/play-slick/blob/d4ecda8f38928f7c56e4ab78659d9aae56e82dce/build.sbt#L25
* https://github.com/playframework/playframework/blob/f18c239c9186b5b65af291817cd75918f5d39ff9/project/Release.scala#L17 (via https://github.com/playframework/playframework/blob/f18c239c9186b5b65af291817cd75918f5d39ff9/build.sbt#L482)

Now... scalatestplus-play and slick are not sbt-plugins anyway but, in interplay terms, libraries and therefore set their cross versions to 2.12 and 2.13. Play just sets it to empty list for the root project.

The conclusion for me is that `PlayRootProject` was meant to be used for "libraries" rather than sbt plugins and you don't really want to enable `PlayRootProject` if your project is an sbt plugin, but if you do, you want to set cross version explicitly.